src/main/Java/SampleApp.java

```diff
@@ -0,0 +1,267 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.util.Properties;
+import java.util.Scanner;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.http.HttpVersion;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.entity.ContentType;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.CDL;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+public class SampleApp {
+	private static final String DEFAULT_PROPERTIES_FILE = "environment.properties";
+	private static final Logger logger = LogManager.getLogger(SampleApp.class);
+	private static Properties props;
+	
+	/**
+	 * This application makes sample API calls to the CHPL API:
+	 * - shows how to make API calls using org.apache.http
+	 * - shows how to parse the JSON response using com.google.gson.jsonparser
+	 */
+	public SampleApp(){}
+	
+	public static void main(String[] args) throws IOException, URISyntaxException, JSONException {
+		SampleApp sampleApp = new SampleApp();
+		String token = null;
+		
+		props = sampleApp.loadProperties(DEFAULT_PROPERTIES_FILE);
+		sampleApp.displayInstructions();
+		Integer input = sampleApp.getValidUserInput();
+		
+		/**
+		 * Example 1:
```

Comment:
I think these comments would be better placed at the `sampleApp.getToken()` definition. Here it looks like the steps do three things, but the next lines don't show those things